internal/jujuapi/cloud.go

```diff
@@ -276,30 +277,18 @@ func collapseUpdateCredentialResults(args jujuparams.TaggedCredentials, updateRe
 	return results
 }
 
-func userModelAccess(ctx context.Context, user *openfga.User, model names.ModelTag) (string, error) {
-	isModelAdmin, err := openfga.IsAdministrator(ctx, user, model)
-	if err != nil {
-		return "", errors.E(err)
-	}
-	if isModelAdmin {
-		return "admin", nil
-	}
-	hasWriteAccess, err := user.IsModelWriter(ctx, model)
-	if err != nil {
-		return "", errors.E(err)
-	}
-	if hasWriteAccess {
-		return "write", nil
-	}
-	hasReadAccess, err := user.IsModelReader(ctx, model)
-	if err != nil {
-		return "", errors.E(err)
-	}
-	if hasReadAccess {
-		return "read", nil
+func userModelAccess(ctx context.Context, user *openfga.User, model names.ModelTag) string {
+	userRelation := user.GetModelAccess(ctx, model)
+	switch userRelation {
+	case ofganames.AdministratorRelation:
```

Comment:
I know we have ofganames now, but I really think permission package with access constants would've been better from juju.